modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -69,7 +69,8 @@ module.exports = ({
       alias: {
         vue$: 'vue/dist/vue.runtime.esm.js',
         // resolve apostrophe modules
-        Modules: path.resolve(modulesDir)
+        Modules: path.resolve(modulesDir),
+        Root: path.resolve(__dirname, '../../../../../..')
```

Comment:
An interesting idea, but I think `import { something} from 'apostrophe/lib/whatever.js'` is just as effective, labels what is happening more clearly, and doesn't require adding any new webpack configuration.

Reply:
(That is, in the frontend build, importing stuff from sub-files of `apostrophe` is just as valid as importing stuff from sub-files of any other npm module.)

Reply:
That is way better indeed, thanks!